### DISCUSSION TITLE
FedRamp Questions

### DISCUSSION THREAD

**Discussion by jgb1128** [CC-in-the-Cloud/Admin#69] [General]
Log FedRamp questions here

**Comment by e-newt** [comment]
Applications could use FR Tailored menus, correct?
Further, a question to ask is if ancellary cloud services, such as Apple Business Manager (used to obtain volume purchasing program licensing for apps and then connect to MDMs) need some sort of a certification/trust?

**Reply by e-newt** [reply]
Probably no

**Comment by mbdowne** [comment]
How easy is it to prohibit multi-tenancy in a given certification?

**Reply by e-newt** [reply]
Sam L from PMO: data is required to be separated at least logically. Multi-tenancy is allowed.... The customer drives this.

**Comment by e-newt** [comment]
#70

**Comment by e-newt** [comment]
Does FedRAMP have a concept of equivalency?

**Reply by e-newt** [reply]
From discussion with Sam Leestma with the FedRAMP PMO:A security impact analysis is used to determine if a change is just an update or more.

**Reply by e-newt** [reply]
GSA policy: A change in infrastructure would be considered a significant change that would need to be evaluated for the scope of the change, impact on the risk posture, and could possibly result in the need for re-authorization. See the FedRAMP Program Management Office’s (PMO’s) [Significant Change Policies and Procedures guidance [WORD - 563KB]] ([https://www.fedramp.gov/assets/resources/documents/CSP_Significant_Change_Policies_and_Procedures.docx]) for more information.
 
https://www.fedramp.gov/assets/resources/documents/CSP_Significant_Change_Policies_and_Procedures.docx

**Comment by e-newt** [comment]
Will GSA/FedRAMP be able to accept CC documentation or testing (for specific requirements)?

**Reply by e-newt** [reply]
Paraphrasing Sam L from the PMO: FedRAMP 3PAO has to validate, but documentation might be reusable.

**Reply by e-newt** [reply]
Too soon to ask, but look at the forthcoming Rev 5 guidance, which may allow some alternatives.

Boundary guidance will be loosened.

**Comment by e-newt** [comment]
To have FedRAMP participate in this activity, formally contact Brian Conrad w/ the TTS Group.

**Reply by e-newt** [reply]
Action for @GY5318 / Greg Youst-- Greg will work with his colleagues at DISA (FedRAMP reviewers) to invite Brian.
